### DISCUSSION TITLE
Implementation web site with a directory in GitHub

### DISCUSSION THREAD

**Discussion by ElleWilte** [community/community#32921] [Other Feature Feedback, Questions, & Ideas]
My website contains a directory (huiswerk_les9) with the home page and other directories. With this link you can see the structure of the website: 'https://github.com/ElleWilte/huiswerkles9/tree/main/huiswerk_les9'. To open the site, you have to start with the page home. Without GitHub, the website works fine. However, the URL 'https://ellewilte.github.io/huiswerkles9/huiswerk_les9/home.htm' generates the error-message '404 There isn't a GitHub Pages site here'. Please help me, to implement that web page in GitHub. Many thanks in advance, Elle Evenhuis

**Comment by airtower-luna** [comment]
You don't seem to have Pages enabled for the repository, otherwise there should be a "build and deploy" workflow in the Actions tab. Try enabling it in the repository settings and wait a little. :slightly_smiling_face:
